docs/UserGuide.md

```diff
@@ -362,14 +362,16 @@ Usage:
 Example:
 - After all this talk about Aglio Olio and you've yet to view the details of the recipe? Don't worry, all you have to do is run
     - `view Aglio Olio`
-      - This would display the full contents of the recipe matching the name "Aglio Olio" including its ingredients and steps to prepare the dish.
+      - This would display the full contents of the recipe matching the name "Aglio Olio" including the ingredients and steps to prepare the dish.
       
 - Aglio Olio is the first recipe? Even better! You just have to run:
     - `view -x 1`
-      - This would display the full contents of the first recipe in the list which includes its ingredients and steps to prepare the dish.
+      - This would display the full contents of the first recipe in displayed list including the ingredients and steps to prepare the dish.
```

Comment:
```suggestion
      - This would display the full contents of the first recipe in the displayed list including the ingredients and steps to prepare the dish.
```